101-vm-linux-Jupyterhub/scripts/install.sh

```diff
@@ -0,0 +1,3 @@
+!/bin/bash
+
+curl https://raw.githubusercontent.com/jupyterhub/the-littlest-jupyterhub/master/bootstrap/bootstrap.py | sudo python3 - --admin $1 >> /var/log/TLJH_install.log 2>&1
```

Comment:
this needs to be included in the sample as well

Reply:
bootstrap.py needs to be included in this sample

Reply:
@jose-mart  so simply change the install.sh to install bootstrap.py locally so add bootstrap.py to the scripts folder and add the following install.sh

!/bin/bash
bootstrap.py | sudo python3 - --admin $1 >> /var/log/TLJH_install.log 2>&1

Reply:
Included